docs/latest/essentials/translating.md

```diff
@@ -115,9 +115,9 @@ When needed, you can also use available translations _without React_.
 E.g.
 
 ```js
-import { t } from '@ohif/i18n';
-console.log(t('my translated text'));
-console.log(t('$t(Common:Play) my translated text'));
+import { T } from '@ohif/i18n';
```

Comment:
Why?

Reply:
When I had put `t` in the `i18n` object to be exported, I got `Maximum call stack size exceeded.` -- maybe some conflict with the standard `t`.

I just changed to `T` and the error has gone. Just to be clear, this `T` is related to the use of the lib without react: https://docs.ohif.org/essentials/translating.html#without-react
It’s not the same `t` we are getting with HOCs.